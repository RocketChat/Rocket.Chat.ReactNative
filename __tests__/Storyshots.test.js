import initStoryshots from '@storybook/addon-storyshots';

jest.mock('../app/lib/database', () => jest.fn(() => null));
global.Date.now = jest.fn(() => new Date('2019-10-10').getTime());

initStoryshots();
