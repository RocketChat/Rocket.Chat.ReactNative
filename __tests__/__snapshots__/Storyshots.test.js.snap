// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Avatar avatar 1`] = `
<RCTScrollView>
  <View>
    <View
      style={
        Array [
          Object {
            "borderRadius": 4,
            "height": 25,
            "width": 25,
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 4,
              "height": 25,
              "width": 25,
            },
          ]
        }
      >
        <FastImageView
          resizeMode="cover"
          source={
            Object {
              "priority": "high",
              "uri": "baseUrl/avatar/test?format=png&width=50&height=50",
            }
          }
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "borderRadius": 4,
            "height": 40,
            "width": 40,
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 4,
              "height": 40,
              "width": 40,
            },
          ]
        }
      >
        <FastImageView
          resizeMode="cover"
          source={
            Object {
              "priority": "high",
              "uri": "baseUrl/avatar/aa?format=png&width=50&height=50",
            }
          }
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "borderRadius": 4,
            "height": 30,
            "width": 30,
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 4,
              "height": 30,
              "width": 30,
            },
          ]
        }
      >
        <FastImageView
          resizeMode="cover"
          source={
            Object {
              "priority": "high",
              "uri": "baseUrl/avatar/bb?format=png&width=50&height=50",
            }
          }
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "borderRadius": 2,
            "height": 25,
            "width": 25,
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            Object {
              "borderRadius": 2,
              "height": 25,
              "width": 25,
            },
          ]
        }
      >
        <FastImageView
          resizeMode="cover"
          source={
            Object {
              "priority": "high",
              "uri": "baseUrl/avatar/test?format=png&width=50&height=50",
            }
          }
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        />
      </View>
    </View>
  </View>
</RCTScrollView>
`;

exports[`Storyshots Channel Cell Direct Messages 1`] = `
<RCTScrollView>
  <View>
    View
    View
    View
    View
    View
    View
    View
    View
    View
    View
    View
  </View>
</RCTScrollView>
`;

exports[`Storyshots Message list 1`] = `
<RCTScrollView
  contentContainerStyle={
    Object {
      "marginVertical": 30,
    }
  }
  style={
    Object {
      "flex": 1,
      "transform": Array [
        Object {
          "scaleY": -1,
        },
      ],
    }
  }
>
  <View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Simple
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Long message
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Grouped messages
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Without header
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      With alias
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Edited
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Static avatar
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Full name
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Mentions
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Emojis
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Custom Emojis
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Time format
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Reactions
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Multiple reactions
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Intercalated users
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "marginBottom": 25,
          "marginTop": 15,
          "transform": Array [
            Object {
              "scaleY": -1,
            },
          ],
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#9ea2a8",
              "fontSize": 14,
              "fontWeight": "600",
            },
            Object {
              "color": "#f5455c",
            },
            Object {
              "marginLeft": 10,
            },
          ]
        }
      >
        Nov 10, 2017
      </Text>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#9ea2a8",
              "flex": 1,
              "height": 1,
            },
            Object {
              "backgroundColor": "#f5455c",
            },
            Object {
              "marginHorizontal": 10,
            },
          ]
        }
      />
      <Text
        style={
          Array [
            Object {
              "color": "#9ea2a8",
              "fontSize": 14,
              "fontWeight": "600",
            },
            Object {
              "color": "#f5455c",
            },
            Object {
              "marginRight": 10,
            },
          ]
        }
      >
        unread messages
      </Text>
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "marginBottom": 25,
          "marginTop": 15,
          "transform": Array [
            Object {
              "scaleY": -1,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#9ea2a8",
              "flex": 1,
              "height": 1,
            },
            Object {
              "backgroundColor": "#f5455c",
            },
            Object {
              "marginLeft": 10,
            },
          ]
        }
      />
      <Text
        style={
          Array [
            Object {
              "color": "#9ea2a8",
              "fontSize": 14,
              "fontWeight": "600",
            },
            Object {
              "color": "#f5455c",
            },
            Object {
              "marginHorizontal": 10,
            },
          ]
        }
      >
        unread messages
      </Text>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#9ea2a8",
              "flex": 1,
              "height": 1,
            },
            Object {
              "backgroundColor": "#f5455c",
            },
            Object {
              "marginRight": 10,
            },
          ]
        }
      />
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "marginBottom": 25,
          "marginTop": 15,
          "transform": Array [
            Object {
              "scaleY": -1,
            },
          ],
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#9ea2a8",
              "flex": 1,
              "height": 1,
            },
            Object {
              "marginLeft": 10,
            },
          ]
        }
      />
      <Text
        style={
          Array [
            Object {
              "color": "#9ea2a8",
              "fontSize": 14,
              "fontWeight": "600",
            },
            Object {
              "marginHorizontal": 10,
            },
          ]
        }
      >
        Nov 10, 2017
      </Text>
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#9ea2a8",
              "flex": 1,
              "height": 1,
            },
            Object {
              "marginRight": 10,
            },
          ]
        }
      />
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Date and Unread separators
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      With image
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      With video
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      With audio
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Message with reply
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      URL
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Custom fields
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Two short custom fields
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Broadcast
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Archived
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
      View
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Error
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Temp
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Editing
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Removed
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Joined
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Room name changed
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Message pinned
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Has left the channel
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      User removed
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      User added
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      User muted
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      User unmuted
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Role added
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Role removed
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Changed description
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Changed announcement
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Changed topic
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Changed type
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Custom style
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Markdown emphasis
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Markdown headers
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Markdown links
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Markdown image
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Markdown code
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Markdown quote
    </Text>
    <View
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      View
    </View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 20,
            "fontWeight": "300",
            "marginLeft": 10,
            "marginTop": 30,
          },
          Object {
            "marginBottom": 30,
            "marginTop": 0,
            "transform": Array [
              Object {
                "scaleY": -1,
              },
            ],
          },
        ]
      }
    >
      Markdown table
    </Text>
  </View>
</RCTScrollView>
`;
